lmctl/utils/__init__.py

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Could this go in a "file" or "certificate" module of utils?

Reply:
Yep